ibis/backends/duckdb/registry.py

```diff
@@ -28,6 +28,18 @@
 )
 from ibis.common.exceptions import UnsupportedOperationError
 
+try:
+    from geoalchemy2 import Geometry
+
+    class Geometry_WKB(Geometry):
+        as_binary = "ST_AsWKB"
+
+except ImportError:
+
+    class Geometry_WKB:
+        ...
+
+
```

Comment:
I think we can import `Geometry_WKB` from `ibis.backends.duckdb.datatypes` and save the copy-paste here (that I did) -- and then one day when we hopefully rip this workaround out we only have to remove it in one spot.